rotkehlchen/exchanges/coinbase.py

```diff
@@ -702,11 +715,118 @@ def query_online_deposits_withdrawals(
             movement = self._deserialize_asset_movement(raw_movement)
             # limit coinbase deposit/withdrawals in the requested time range
             #  here since there is no argument in the API call
-            if movement and movement.timestamp >= start_ts and movement.timestamp <= end_ts:
+            if movement and start_ts <= movement.timestamp <= end_ts:
                 movements.append(movement)
 
         return movements
 
+    def _deserialize_ledger_action(self, raw_data: Dict[str, Any]) -> Optional[LedgerAction]:
+        """Processes a single transaction from coinbase and deserializes it
+
+        Can log error/warning and return None if something went wrong at deserialization
+        This method can raise:
+        - DeserializationError
+        - UnknownAsset
+        - UnsupportedAsset
```

Comment:
I don't think it can. It seems to all be surrounded in a try/except which actually catches and handles those.